apps/web/src/actions/latte/stopChat.ts

```diff
@@ -0,0 +1,49 @@
+'use server'
+
+import { z } from 'zod'
+import { authProcedure } from '$/actions/procedures'
+import { cancelJob } from '@latitude-data/core/services/bullmq/cancelJob'
+import { documentsQueue } from '@latitude-data/core/queues'
+
+const REMOVABLE_STATES = [
+  'completed',
+  'failed',
+  'delayed',
+  'waiting',
+  'waiting-children',
+]
+const NON_REMOVABLE_STATES = ['active', 'stalled']
+
+export const stopChatLatteAction = authProcedure
+  .createServerAction()
+  .input(
+    z.object({
+      bullMQjobId: z.string().optional(),
```

Comment:
No need to change, but better not to let traces of the underlying tech. If tomorrow we swap the jobs queues system from bullMQ to something else, this won't make sense. 

Just `jobId` is better